config/dictionaries/resource.json

```diff
@@ -928,7 +922,9 @@
     "fields": [
       "id",
       "actor",
-      "username"
+      "username",
+      "created_at",
+      "has_legacy_link_to__organization"
```

Comment:
Let's drop this one.

Reply:
Dropping `has_legacy_link_to__organization`